src/Administration/Resources/app/administration/src/app/component/form/sw-custom-field-set-renderer/index.js

```diff
@@ -326,14 +335,71 @@ Component.register('sw-custom-field-set-renderer', {
             };
         },
 
-        onChangeCustomFieldSets(value, updateFn) {
+        customFieldSetCriteriaById() {
+            const criteria = new Criteria(1, 1);
+            const customFieldsCriteria = new Criteria(1);
+
+            customFieldsCriteria.addSorting(Criteria.sort('config.customFieldsPosition'));
+
+            criteria
+                .addAssociation('customFields', customFieldsCriteria);
+
+            return criteria;
+        },
+
+        loadCustomFieldSet(setId) {
+            if (this.loadingFields.includes(setId)) {
+                // as we might triggered multiple times with the same item, we store the loading set in a heap cache
+                return;
+            }
+
+            const set = this.sets.find(s => s.id === setId);
```

Comment:
you can use the entity collection functions here: https://github.com/shopware/admin-extension-sdk/blob/main/src/data/_internals/EntityCollection.ts

`this.sets.get()`

Reply:
done